src/configure/model/templateModels.ts

```diff
@@ -1,26 +1,44 @@
-import { ExtendedPipelineTemplate } from "./Contracts";
 import { AzureConnectionType, ServiceConnectionType, TargetKind, TargetResourceType } from "./models";
 
+export enum TemplateType {
+    remote,
+    local
+}
 export interface PipelineTemplate {
-    path: string;
     label: string;
-    language: string;
+    templateWeight: number;
+    templateType: TemplateType;
```

Comment:
I missed to find where this type is being set. Can you please point me out?

Reply:
For local templates, it's defined in the template. For remote, we are setting it in templateHelper.ts.
```
export function getTargetType(template: PipelineTemplateMetadata) {
    if (template.attributes.deployTarget.toLowerCase().includes("webapp")) {
        return TargetResourceType.WebApp;
    } else if (template.attributes.deployTarget === "Azure:AKS") {
        return TargetResourceType.AKS;
    }
    return TargetResourceType.None;
}
```

Reply:
My question was for templateType and not targetType